commands/hexpire.md

```diff
@@ -0,0 +1,33 @@
+The `HEXPIRE` command allows manipulating the hash expiration time for existing fields.
+When set, the expiration time of a hash field will determine when the field will be automatically reclaimed.
```

Comment:
Is it "hash expiration of fields" or "expiration of hash fields"? The latter sounds better to me.

The word "reclaimed" looks odd here. The memory can be reclaimed to be used for other things, but the expired hash field itself is ... deleted?

```suggestion
The `HEXPIRE` command allows manipulating the expiration time for existing hash fields.
When set, the expiration time of a hash field will determine when the field will be automatically deleted.
```